utility/cancel_requests_x_days_old

```diff
@@ -0,0 +1,219 @@
+<script src="../libs/js/LEAF/intervalQueue.js"></script>
+<script>
+    var numDays = 30;
```

Comment:
Are we sure this variable needs to be global?

Reply:
I think this is appropriate since this script is meant to do one thing, and a developer can easily change this if needed.

Reply:
Yes, it's used inside a function further down the page.